pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -904,13 +904,41 @@ func (tikv *TiKVSpec) GetRecoverByUID() types.UID {
 	return tikv.Failover.RecoverByUID
 }
 
+func (tikv *TiKVSpec) GetScaleInParallelism() int {
+	if tikv.ScaleInParallelism == nil {
+		return 1
+	}
+	return int(*(tikv.ScaleInParallelism))
```

Comment:
Is `0` a valid value?

Reply:
https://github.com/pingcap/tidb-operator/pull/3881#discussion_r603750804

For a very very long time ago, we had a discussion here.

Reply:
OK, sorry for the confusion.